src/topp/FeatureLinkerBase.cpp

```diff
@@ -292,12 +295,21 @@ class TOPPFeatureLinkerBase :
         StringList ms_runs;
         maps[i].getPrimaryMSRunPath(ms_runs);
         ms_run_locations.insert(ms_run_locations.end(), ms_runs.begin(), ms_runs.end());
+        if (keep_subelements)
+        {
+          std::function<void (PeptideIdentification &)> saveOldMapIndex = [](PeptideIdentification& p)
```

Comment:
for these cases I prefer auto

Reply:
It somehow doesn't work with the std::function wrapper.